LaTeX/TeX.sublime-syntax

```diff
@@ -277,30 +272,51 @@ contexts:
     - include: paragraph-pop
 
   def-definition:
-    - match: \{
-      scope: punctuation.definition.group.brace.begin.tex
-      set: def-definition-body
+    - match: (?=\{)
+      set:
+        - def-scope-as-body
+        - macro-braces-begin
     - include: paragraph-pop
     - include: else-pop
 
-  def-definition-body:
-    - meta_scope: meta.function.body.tex meta.group.brace.tex
-    - include: macro-brace-group-body
+  def-scope-as-body:
+    - meta_scope: meta.function.body.tex
+    - include: immediately-pop
```

Comment:
Contexts which are meant to apply meta_scopes only should exclude prototypes to prevent inheriting syntaxes to inject code via `prototype` context.

```suggestion
  def-scope-as-body:
    - meta_include_prototype: false
    - meta_scope: meta.function.body.tex
    - include: immediately-pop
```

Not sure about the context name. Such meta_scope only contexts are often named `...-meta`. Maybe something like `def-body-meta` would be suitable?